frontend_v2/src/App.js

```diff
@@ -0,0 +1,65 @@
+import logo from './tqec_logo.svg';
+import './App.css';
+import {Stage} from '@pixi/react'
+import TqecApp from './workspace'
+
+/* The logo has been created as the SVG rendering of the ASCII:
+ *
+ *    /---/  //---/    //---/   //---/
+ *     //   //   /    //-/     //
+ *    //   //---/==  //---/   //---/
+ *
+ *  with the web app: https://ivanceras.github.io/svgbob-editor/
+ */
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <a
+          className="App-link"
+          href="https://github.com/QCHackers/tqec"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          TQEC repo
+        </a>
+      </header>
+
+      <div className="Tabs">
+        <script>console.log('test log from App.js (tabs)')</script>
```

Comment:
These `console.log` statements are not needed.

Reply:
They were for debugging and some have "survived"...
Thanks for catching them!